COMPREDICT-GmbH/ai-sdk-python
# 35 split features and parameters

Removed: 
	- global variable ACCEPTED CONTENT TYPES
	- content_type arg from run_algorithm and train_algorithm
	- adjust_file_name_to_content_type utils function
	- tests associated

Added: 
	- additional 'parameters' arg in run_algorithm and train_algorithm
	- __raise_error_if_file_type_incorrect function to check if features file is of type parquet and parameters file
is of type json
	- in get_template and get_graph add information for the user about 'parameters' type

Refactored:
	- extract method remove_file() from 'finally' blocks in run_algorithm and train_algorithm()
	- remove 'except' block in run_algorithm and train_algorithm as redundunt
	- rename 'data' as it is quite broad arg name to 'features' and allow features to be of type list as well
	- change process_data method to process features as well as parameters (had doubts if better to deal with higher complexity of the code or rather create two different methods from this one, like: process_features and process_parameters, I solved this with, hopefully, quite clear docs, but still not sure (?)
	- include parameters in docs and make some other parts of docs clearer
	- refactor tests accroding to changes in the code, add two testing files to media folder, necessary for testing
